src/sentry/api/endpoints/project_group_index.py

```diff
@@ -164,6 +165,23 @@ def _build_query_params_from_request(self, request, project):
 
         return query_kwargs
 
+    def _subscribe_and_assign_issue(self, acting_user, group, result):
+        if acting_user:
+            GroupSubscription.objects.subscribe(
+                user=acting_user,
+                group=group,
+                reason=GroupSubscriptionReason.status_change,
+            )
+            self_assign_issue = UserOption.objects.get_value(
+                user=acting_user,
+                project=None,
+                key='self_assign_issue',
+                default='0'
+            )
+            if not group.assignee_set.exists() and self_assign_issue == '1':
+                result['assignedTo'] = extract_lazy_object(acting_user)
+        return result
```

Comment:
This is a minor stylistic nitpick, but since this method mutates the state of the arguments, I don't think we need to return a value here. For a example of precedent here, `dict.update` and `list.append` and other standard library in-place mutation methods don't return any values.